.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/locales/translations"]
 	path = src/locales/translations
-	url = https://github.com/OlympusDAO/olympus-translations
+	url = https://github.com/OlympusDAO/olympus-translations.git
 	branch = develop
```

Comment:
do you want to change this to `main`

Reply:
Ok but in this case we need a script to switch to develop, so we can preview translations on staging area.

Reply:
Added a commit that does that